src/other/tables.jl

```diff
@@ -21,7 +21,7 @@ end
 Tables.columnindex(df::Union{AbstractDataFrame, DataFrameRow}, idx::AbstractString) =
     columnindex(df, Symbol(idx))
 
-Tables.schema(df::AbstractDataFrame) = Tables.Schema{Tuple(_names(df)), Tuple{[eltype(col) for col in eachcol(df)]...}}()
+Tables.schema(df::AbstractDataFrame) = Tables.Schema(_names(df), [eltype(col) for col in eachcol(df)])
```

Comment:
`_names(df)` returns a `SubArray` if `df` is a `SubDataFrame`. Is this acceptable by `Tables.schema`?

Also `_names(df)` for a `DataFrame` returns an internal vector (without copying). Is this safe?

In short. Maybe it should be ` propertynames(df)` instead (which guarantees a `Vector{Symbol}` that is always a copy)

Reply:
Nah, Tables.jl will make its own copy; it accepts any iterable and makes its own `Vector{Symbol}`.

Reply:
Then I just "approve" :).